packages/core/src/codewhispererChat/constants.ts

```diff
@@ -19,3 +21,10 @@ export const getUserPromptsDirectory = () => {
 }
 
 export const createSavedPromptCommandId = 'create-saved-prompt'
+
+export const tools: Tool[] = Object.entries(toolsJson).map(([, toolSpec]) => ({
+    toolSpecification: {
+        ...toolSpec,
+        inputSchema: { json: toolSpec.inputSchema },
```

Comment:
nit: I think we don't need to do this anymore because we changed the tool index from `index_schema` to `indexSchema`